app/lib/draggable.spec.js

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import sinon from 'sinon';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import { Draggable } from './draggable';
+
+describe('Draggable', function () {
+  let wrapper;
+  let handleDrag;
+  let handleEnd;
+  let onStart;
+  let onDrag;
+  let onEnd;
+  before(function () {
+    onStart = sinon.stub();
+    onDrag = sinon.stub().callsFake((e, d) => d);
+    onEnd = sinon.stub();
+    wrapper = shallow(
+      <Draggable
+        onStart={onStart}
+        onDrag={onDrag}
+        onEnd={onEnd}
+      >
+        <p>Hello</p>
+      </Draggable>
+    );
+  });
+  describe('on mousedown', function () {
```

Comment:
These tests get repeated three times for mouse, touch and pointer. They could be rolled up into a function that's called three times with different event types.